api/src/deps.py

```diff
@@ -3,7 +3,6 @@
 
 import aioboto3 as aioboto3
```

Comment:
Лучше эту строку изменить на 
```suggestion
import aioboto3
```